stats/stats-server/src/server.rs

```diff
@@ -56,6 +59,85 @@ fn grpc_router<S: StatsService>(
         .add_service(StatsServiceServer::from_arc(stats))
 }
 
+fn is_threshold_passed(
+    threshold: Option<f32>,
```

Comment:
I believe it make sense to make `threshold` we set during configuration also `f64`, considering we parse api response to `f64` value